index.js

```diff
@@ -35,6 +35,11 @@ export const buildList = () => {
       if (s.includes('red')) return
       if (s.includes('bluegray')) return
       if (s.includes('gray')) return
+      if (s.includes('focus-ring')) return
+      if (s.includes('bg-current')) return
+      if (s.includes('bg-white')) return
+      if (s.includes('bg-transparent')) return
+      if (s.includes('bg-none')) return
```

Comment:
Does these make sense here? I think we will handle the background colors with a help of tokens. But please correct me if I'm wrong

Reply:
`bg-none` sounds though like a different thing

Reply:
Yeah we might have a `bg-none`, but the rest are OK for now - we might need to iterate on these anyhow so no big stress!